.travis.yml

```diff
@@ -35,6 +35,25 @@ jobs:
         # Travis will install requirements.txt by default
         - flake8 --select=E121,E123,E126,E226,E24,E704,W503,W504 --ignore=E501 app.py tests
 
+    # the build process from James Eddy adapted from the WES repo
```

Comment:
Just a note: the steps/stages described under `jobs` should be able to replace the `python` and `stages` sections above. Might be a task for a separate PR.